src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -77,6 +113,7 @@ private void AddOrRemoveDropdown(bool enabled)
                 var document = _languageService.EditorAdaptersFactoryService.GetDataBuffer(buffer).AsTextContainer().GetRelatedDocuments().FirstOrDefault();
                 if (document.GetLanguageService<INavigationBarItemService>() == null)
                 {
+                    RemoveDropdownBar(dropdownManager);
```

Comment:
Does RemoveDropdownBar need to check that _navigationBarController is non-null now _before_ calling dropdownManager.RemoveDropdownBar? My worry here being if this reruns do we risk removing somebody *else's* drop down? (like, say LSPs?)

Reply:
True actually - the dropdown manager may not be ours.  Will check that good catch.